regcomp.h

```diff
@@ -216,13 +216,13 @@ struct regnode_2L {
     I32 arg2;
 };
 
-/* 'Two field' -- Two 16 bit unsigned args */
+/* 'Two field' -- Two 32 bit signed args */
```

Comment:
I hadn't really thought about it. But yeah, probably you are right, we can merge 2L and 2 together, and use a union to have one argument be signed. I am not sure I want to deal with it until most of the important parts of the patch sequence are resolved. 

Related to this is that I would like to get rid of BRANCHJ and LONGJMP as they wont be needed anymore by the time this PR sequence is done.

Reply:
I pushed a patch to the end of the sequence which merges the regnode_2L and regnode_2 into a single struct, and introduces a consistent set of macros to access fields called "arg". With this patch we have a regnode_1, regnode_2, regnode_3, each of which can hold one more 32 bit argument as the previous. This storage is set up so it can store both U32 and I32 values, or two U16 values. Macros of the form ARG1i() ARG1u() and ARG1a() and ARG1b() can be used to access each arg field.  ANy other regnode types that also had arg1 or arg2 fields received the same treatment.